### PR TITLE
doc - add content for new greenplum_fdw module

### PR DESCRIPTION
**Pull request by lisakowen**
add some basic module docs for greenplum_fdw.  this is based on postgres_fdw.  the org of this page differs from the org of postgres_fdw docs.  in some cases i xref'd to the postgres docs.

doc review site link (behind vpn):  https://docs-lisa-gpfdw.sc2-04-pcf1-apps.oc.vmware.com/6-19/ref_guide/modules/greenplum_fdw.html

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/modules/greenplum_fdw.xml [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+            <xref href="https://www.postgresql.org/docs/9.4/postgres-fdw.html"
+              scope="external">Connection Options</xref> topic in the PostgreSQL
+            <codeph>postgres_fdw</codeph> documentation for more information about
+            these options.</p>
+          <p>To obtain the full benefits of the parallel transfer feature provided by
+            <codeph>greenplum_fdw</codeph>, you must also specify:</p>
+            <codeblock>mpp_execute 'all segments'</codeblock>
+          <p>and</p>
+            <codeblock>num_segments '<varname>num</varname>'</codeblock>
+          <p>in the <codeph>OPTIONS</codeph> clause when you create the server. Set
```
Comment (huiliang-liu) [inline gpdb-doc/dita/ref_guide/modules/greenplum_fdw.xml:100]: `num_segments`''s default value will be the segment number of local greenplum when not specified.

File: gpdb-doc/dita/ref_guide/modules/greenplum_fdw.xml [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+            foreign table. The <codeph>column_name</codeph> option identifies the name of
+            the associated column in the remote Greenplum Database table, and defaults to
+            the foreign table column name when not specified.</p>
+        </section>
+        <section id="query_ftable">
+          <title>Constructing and Running Queries</title>
+          <p>You <codeph>SELECT</codeph> from a foreign table to access the data
+            stored in the underlying remote Greenplum Database table. By default, you can
+            also modify the remote table using <codeph>INSERT</codeph>,
+            <codeph>UPDATE</codeph>, or
```
Comment (huiliang-liu) [inline gpdb-doc/dita/ref_guide/modules/greenplum_fdw.xml:221]: greenplum fdw does't support update and delete, if the `mpp_execute` option is set as `all segments`.

File: gpdb-doc/dita/ref_guide/modules/greenplum_fdw.xml [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+      <title>About the Updatability Option</title>
+      <body>
+        <p>By default, all foreign tables created with <codeph>greenplum__fdw</codeph> are
+          assumed to be updatable. You can override this for a foreign server or a foreign
+          table using the following option:</p>
+        <dl>
+        <dlentry>
+        <dt><codeph>updatable</codeph></dt>
+        <dd>Controls whether <codeph>greenplum_fdw</codeph> allows foreign tables
+          to be modified using <codeph>INSERT</codeph>, <codeph>UPDATE</codeph>, and
```
Comment (huiliang-liu) [inline gpdb-doc/dita/ref_guide/modules/greenplum_fdw.xml:257]: same as above

**Comment by lisakowen** [comment]
thank you for reviewing, @huiliang-liu .  i incorporated your requested edits.  i removed update/delete from supported update operations, and added a related limitation.

**Comment by lisakowen** [comment]
@adam8157 - are you interested in reviewing the greenplum_fdw docs?

File: gpdb-doc/dita/ref_guide/modules/greenplum_fdw.xml [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+    <title>Additional Information</title>
+    <body>
+      <p>For more information about <codeph>greenplum_fdw</codeph> updatability and cost
+        estimation options, connection management, and transaction management, refer to
+        the individual topics below.</p>
+    </body>
+    <topic id="topic_update">
+      <title>About the Updatability Option</title>
+      <body>
+        <p>By default, all foreign tables created with <codeph>greenplum__fdw</codeph> are
```
Comment (adam8157) [inline gpdb-doc/dita/ref_guide/modules/greenplum_fdw.xml:251]: redundant underscore here

**Review by adam8157 — COMMENTED** [review]
Others look good to me, and Huiliang knows greenplum_fdw much better than me. 😊

**Review by huiliang-liu — APPROVED** [review]

**Pull request merged**